state-transition/core/state_processor_validators.go

```diff
@@ -21,53 +21,147 @@
 package core
 
 import (
+	stdbytes "bytes"
+	"fmt"
+	"slices"
+
+	"github.com/berachain/beacon-kit/config/spec"
 	"github.com/berachain/beacon-kit/primitives/bytes"
 	"github.com/berachain/beacon-kit/primitives/math"
 	"github.com/berachain/beacon-kit/primitives/transition"
 	"github.com/sourcegraph/conc/iter"
 )
 
-// processValidatorsSetUpdates returns the validators set updates that
-// will be used by consensus.
+//nolint:lll // let it be
 func (sp *StateProcessor[
 	_, _, _, BeaconStateT, _, _, _, _, _, _, _, _, ValidatorT, _, _, _, _,
-]) processValidatorsSetUpdates(
```

Comment:
this is basically being replace by the following processing:
- at the beginning of `processEpoch` we not the currently active validator set
- we carry out all Eth like transitions
- at the end of `processEpoch` we pick again the active validator set (which will be active next epoch) and compute the diff to send back to consensus